frontend/src/scenes/scenes.ts

```diff
@@ -201,6 +197,7 @@ export const redirects: Record<string, string | ((params: Params) => string)> =
     '/plugins': urls.plugins(),
     '/actions': '/events/actions',
     '/organization/members': urls.organizationSettings(),
+    '/i/:shortId': ({ shortId }) => urls.insightView(shortId),
```

Comment:
where is this coming from? 🤔

Reply:
I removed a separate `insightRouter` logic and component that did this same transformation in a more complicated way.

Reply:
I don't think we need this, as literally nobody uses these links: https://app.posthog.com/insights/1r8zqqtf. We don't show them anywhere in the app and I don't think we have a reason to anymore.

Reply:
I guess we should measure? In any case, I'd keep this for now... just in case.

Reply:
As in, the insight I linked measures that usage – there's none:
<img width="1499" alt="Screen Shot 2022-03-11 at 14 54 18" src="https://user-images.githubusercontent.com/4550621/157880977-eb141799-33d3-43e4-a820-1a7b19a0e032.png">